src/utils/tekton-results.ts

```diff
@@ -333,19 +340,29 @@ export const getTaskRuns = (
 const getLog = (workspace: string, taskRunPath: string) =>
   commonFetchText(`${getTRUrlPrefix(workspace)}/${taskRunPath.replace('/records/', '/logs/')}`);
 
-export const getTaskRunLog = (
+export const getTaskRunLogOld = (
   workspace: string,
   namespace: string,
   taskRunName: string,
 ): Promise<string> =>
   getFilteredRecord<any>(
     workspace,
     namespace,
-    DataType.Log,
+    [DataType.Log, DataType.Log_OLD],
     AND(EQ(`data.spec.resource.kind`, 'TaskRun'), EQ(`data.spec.resource.name`, taskRunName)),
     { limit: 1 },
   ).then((x) =>
     x?.[1]?.records.length > 0
       ? getLog(workspace, x?.[1]?.records[0].name).catch(() => throw404())
       : throw404(),
   );
+
+export const getTaskRunLog = (
+  workspace: string,
+  namespace: string,
+  pid: string,
+  taskRun: TaskRunKind,
+) =>
+  commonFetchText(
+    `${getTRUrlPrefix(workspace)}/${namespace}/results/${pid}/logs/${taskRun?.metadata?.uid}`,
+  ).catch(() => getTaskRunLogOld(workspace, namespace, taskRun.metadata.name));
```

Comment:
Why are you using getTaskRunLogOld?
If it fails to fetch we should just throw error and surface in UI.

Reply:
removed